app/models/comment.rb

```diff
@@ -67,7 +67,7 @@ def parent
     if aid == 0
       node
     else
-      answer.node
+      answer && answer.node
```

Comment:
Here, we have to be careful because `parent` should return the record, not just a boolean. Can you think of how to split this out more clearly? Or maybe we make an alternate way to refer to an answer instead of the parent node?

Reply:
Oops, my mistake, you're right. I just need to check if answer existed then return answer.node

Reply:
I'll make the changes right away!